### ISSUE TITLE
Reorder Start Menu

### ISSUE THREAD

**Issue by Cheyron** [Tyler-IN/MnB2-Bannerlord-CommunityPatch#130]
Would it be possible to move mod options next to game settings?

**Comment by ajseavey** [comment]
I believe moving the mod options into the regular game options is on the road map.

**Comment by Tyler-IN** [comment]
Yeah we're going to move into the Options menu https://github.com/Tyler-IN/MnB2-Bannerlord-CommunityPatch/issues/61

**Comment by Tyler-IN** [comment]
Since someone else is working on the mod options menu, brought focus back around to this.

Have been testing this in the TW Forums - Modding & Mount and Blade discord servers.

I think we got it working well enough. There will be a release for this soon.

**Comment by Tyler-IN** [comment]
From a tester;
![screenshot](https://images-ext-1.discordapp.net/external/7bCFRRP-OluWttcUT8KPMiF7J8o8NBk_E8t4SZf8dcg/https/media.discordapp.net/attachments/333283527363788810/699438829479657522/unknown.png)

We'll be providing a mechanism to prevent mods from forcing other mod options into the unclickable area at the bottom of the menu.

**Comment by Eerye** [comment]
I think the maximum number of menu items that it can still register is 9, from just simply counting the menu items that were working when I had too many mods that added extra mod options to the menu, the 10th item was always unclickable.
